index.html

```diff
@@ -63,6 +81,7 @@ <h3 class="profession">I'am a Software Developer</h3>
     <div class="button1"><button id="button1" type="button">See Project</button>
     </div>
 </section>
+<div class="projects">
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the `See Project` button is only visible when a project is hovered on with a mouse and also the dark opacity should disappear when a project is hovered on with a mouse as seen below

![Screenshot from 2022-03-24 22-18-05](https://user-images.githubusercontent.com/52934128/160012618-0d5b0006-0ed6-4675-a88f-48ceca06d990.png)

Reply:
@topeogunleye Hello, thank you for the feedback, two thoughts:

- I know the pictures do not interact In the same way I thought that would be doing it by using Js. 
- The template is not giving me the background image to those cards, so I guess I need to use some random Image?.

Thank you in advance for your response.

Reply:
Replied through slack